crates/router/src/db/refund.rs

```diff
@@ -389,6 +389,15 @@ mod storage {
                                     pk_id: key.clone(),
                                     source: "refund".to_string(),
                                 },
+                                storage_types::ReverseLookupNew {
```

Comment:
Why merchant_id <> payment_id is lookup id? Already this combination is that pk.